aep/general/0005/aep.md

```diff
@@ -0,0 +1,84 @@
+# Designing an API
+
+This AEP serves as a high-level guide on designing an API that adheres to the
```

Comment:
```suggestion
This AEP serves as a high-level guide to designing an API that adheres to the
```

This sounded a bit off to me, but I wasn't sure if it was just me, so I Googled it and found https://forum.wordreference.com/threads/guide-to-vs-on-vs-for.3645609/.

Separately, the structure of this sentence is a bit ambiguous:  "that adheres to the AEPs" could be describing "an API" or "This AEP".  Maybe something like "...designing an AEP-compliant API"?

Reply:
Sounds good! done.